gdal-sys/build.rs

```diff
@@ -244,11 +242,25 @@ fn main() {
                 "cargo:rustc-cfg=gdal_sys_{}_{}_{}",
                 version.major, version.minor, version.patch
             );
-
+            let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH").expect("Set by cargo");
+            let is_windows = std::env::var("CARGO_CFG_WINDOWS").is_ok();
+            let ptr_size = std::env::var("CARGO_CFG_TARGET_POINTER_WIDTH").expect("Set by cargo");
+
+            let binding_name = match (target_arch.as_str(), ptr_size.as_str(), is_windows) {
```

Comment:
Can we use `env::var("TARGET")` here?

Reply:
I wouldn't want to do that as it makes the match below much longer

As currently written we accept anything that's a 64-bit system independently from the OS as long as it uses `x86_64` ar `aarch64` as architecture. If we would use targets there this would become much more complicated as you would need to all possible target variants, which might be ever changing.